src/app/static/src/tests/components/adr/selectDataset.test.ts

```diff
@@ -351,6 +355,7 @@ describe("select dataset", () => {
         expect(rendered.find(Modal).findAll("button").length).toBe(2);
         expect(rendered.findAll("p").length).toBe(0);
         expect(rendered.find("h4").text()).toBe("Browse ADR");
+        // expectTranslated(rendered.find("h4"), "Browse ADR", "Parcourir ADR", store);
```

Comment:
It doesn't seem to want to pick up the translations on these tests for some reason (I just get English where I should get French). Does anybody have any suggestions?

Reply:
Taking out `sync: false` from the mount seemed to fix it. Not sure why that was in there? French text needs to be updated to 'Checher ADR'

Reply:
I think `sync: false` was needed before the actions were wired up, to stop the component lifecycle running through and closing the modal before loading spinner state could be tested. If this test now passes without it, feel free to remove (and you can probably remove some or all of the `await Vue.nextTick()` in that case.)